playbooks/openstack-update_caasp.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+- hosts: osh-deployer
+  gather_facts: no
+  tasks:
+    - name: Load variables
+      include_vars: "{{ item }}"
+      loop:
+        - "{{ playbook_dir }}/../vars/common-vars.yml"
+        - "{{ playbook_dir }}/../vars/manifest.yml"
+      tags:
+        - always
+
+    - name: Use Velum tooling to update CaaSP admin node
+      shell: "./velum-interactions --update-admin --update-minions -e {{ upstream_repos_clone_folder }}/ccp/environment.json"
```

Comment:
do we need update-minions here? shouldnt that be update-admin only?

Reply:
I splitted it in to tasks in updated commit